test/lib/common.bash

```diff
@@ -238,7 +258,8 @@ checkRemovedPackage() {
 	local pkgbase=$2
 
 	svn up -q "${TMP}/svn-packages-copy/${pkgbase}"
-	if compgen -G "${TMP}/svn-packages-copy/${pkgbase}/repos/${repo}-*" >/dev/null; then
+
+	if __isGlobfile "${TMP}/svn-packages-copy/${pkgbase}/repos/${repo}-"+([^-])"/PKGBUILD"; then
```

Comment:
"While we're at it, just use an array assignment" -- it's not array assignment? It's just glob expansion plus checking that it expanded non-null.